src/assets/styles/variables.scss

```diff
@@ -139,3 +140,6 @@ $font-weight: (
 $sb-avatar-small:                                26px;
 $sb-avatar-normal:                               32px;
 $sb-avatar-large:                                46px;
+
+//Datepicker variables
+$days-borders: 1px solid $light;
```

Comment:
Maybe have use a generic name for this variable, which is used in other components

Reply:
I updated to `light-border`. What do you think?